src/go/rpk/.golangci.yml

```diff
@@ -63,7 +63,7 @@ linters-settings:
   #
   # https://github.com/mvdan/gofumpt/issues/137
   gofumpt:
-    lang-version: "1.17"
+    lang-version: "1.19.2"
```

Comment:
```suggestion
    lang-version: "1.19"
```
not a blocker for merge, but according to [docs](https://pkg.go.dev/mvdan.cc/gofumpt/format#Options), it looks like it is expecting the same `major.minor` format as in the `go.mod` file, but it seems like it is ok to process `major.minor.patch`